### ISSUE TITLE
skipping tests when the repository complies in a way that is not detected by the action

### ISSUE THREAD

**Issue by mkuzak** [fair-software/howfairis#14]
There needs to be a way to specify which test should be skipped. This will be used in a case when the repository owners know that the repository complies with one of the recommendations, but because of the way the checks are done, this is not reflected in the results of the action.
We propose to use a file 'fair-software.yml' that will contain information about tests to skip. Eg. 'SKIP_registry' = TRUE

**Comment by jspaaks** [comment]
For example, ``.howfairis.yml`` could be something like:

```yaml
skip:
  citation
  repository
```

Then the code could do something like 
```python
if "citation" in self.skip.keys():
    self.citation_is_compliant = True
    return self
else:
    # do the checks
```

**Comment by jspaaks** [comment]
Implemented by PR #32

**Comment by jspaaks** [comment]
went with something similar, like so:

```yaml
force:
  repository: true
  license: false
  checklist: false
```

Default file name is ``.howfairis.yml``, can be set with ``-c`` / ``--config`` flag.

Closing this issue.
